benchmarks/compressibility_formulations/.gitignore

```diff
@@ -0,0 +1 @@
+output-*
```

Comment:
Can you please add a README.md in this folder with a sentence what this benchmark is about? Maybe reference the paper?
Also mention that one should run run.sh?